.github/workflows/coordinator-test-v21.yml

```diff
@@ -131,9 +133,9 @@ jobs:
             build-profile: -P dse
             test-profile: -P dse,it-dse-v4-6.8
 
-          - backend: dse-next
-            build-profile: -P dse-next
-            test-profile: -P dse-next,it-dse-next
+#          - backend: dse-next
```

Comment:
Must also comment these, otherwise `dse-next` will be added to matrix (won't remove yet just in case we want to re-enable in near future; if not can remove for good)